build/yaml/pipelines/signed.yaml

```diff
@@ -22,11 +23,17 @@ stages:
       Configuration: Release
       IsMicroBuildInternal: true
 
+- stage: PostBuild
+  jobs:
+  - template: ../jobs/windows/postbuild.yaml 
+    parameters:
```

Comment:
None of these parameters are part of the postbuild.yaml template except SignType (which isn't used).

Reply:
Maybe these were accidentally moved from the packages.yaml template inclusion?

Reply:
Oops, fixing